cartridges/int_bolt_sfra/cartridge/controllers/Login.js

```diff
@@ -42,6 +42,12 @@ server.get('OAuthRedirectBolt', function (req, res, next) {
         renderError(res, next);
     }
 
+    // if shopper login during checkout, set bolt order id to session cache.
+    // update session id to Bolt later in Account-Show since session id changed after login 
```

Comment:
Can you clarify what this line means, in PR description you mention the session id doesn't change even after login.